### PR TITLE
added wikibooks feature

### PR DESCRIPTION
**Pull request by MaxReinisch**

### CODE REVIEW

File: booklist.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <meta content="text/html;charset=utf-8" http-equiv="Content-Type">
+  <meta content="utf-8" http-equiv="encoding">
+  <title>Books</title>
+
+  <link rel="stylesheet" type="text/css" href="css/bootstrap.min.css">
+  <link rel="stylesheet" type="text/css" href="css/font-awesome.min.css">
+  <link rel="stylesheet" href="css/common.css">
```
Comment (vbanos) [inline booklist.html]: Missing `type`

File: css/booklist.css [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+    position: relative;
+    width: auto;
+    display: flex;
+    background-color: rgb(9, 150, 248); color: rgb(255, 255, 255);
+    text-align: left;
+    align-items: center;
+    border-radius: 3px;
+    justify-content: center;
+    text-decoration: none;
+}
```
Comment (vbanos) [inline css/booklist.css]: Aren't these `@font-face`s and `#horizontal_line` already included in `common.css`?

**Comment by vbanos** [comment]
@MaxReinisch there is some commented code all around your PR.
If its not needed, could you please remove it? If its necessary as doc/comment/help for future work, could you also add some comments with that?
There are many empty lines all around, could you trim them to improve the overall code readability?
Thank you.

File: booklist.html (hunk 1 above)
Comment (MaxReinisch) [reply booklist.html]: In my most recent push, I added `type` in this file, as well as in `recommendations.html`

File: settings.html [hunk 3]
```diff
@@ -49,10 +49,20 @@ <h2 style="font-family: Gill Sans Light">Show News Recommendation</h2>
...
+						<div id="show_option_wikibooks"  class="col-sm-3">
+								<div class="wikibooks" style="padding-top: 5px;">
+										<h2 style="font-family: Gill Sans Light">Wikipedia books settings</h2>
+										<h4>Read books cited by Wikipedia</h4>
+										<label for="wikibooks" style="font-size: 14px;">
+												<input type="checkbox" id="wikibooks">
+												<span class="enable"><b>Present "Wikibooks" pop-ups</b></span>
+										</label>
+								</div>
+						</div>
```
Comment (vbanos) [inline settings.html]: Please watch out the code identation, it should be uniform in all the code. Please fix this.

Comment (MaxReinisch) [reply settings.html]: That's weird.  It is uniform in my IDE.  
image

**Comment by vbanos** [comment]
wikibooks branch merged: https://github.com/internetarchive/wayback-machine-chrome/pull/153
I'm closing this.

**Pull request closed without merging**